cmd/celestia/bridge.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-node/cmd"
+	"github.com/celestiaorg/celestia-node/node"
+)
+
+func init() {
+	bridgeCmd.AddCommand(
+		cmd.Init(repoFlagName, node.Bridge),
+		cmd.Start(repoFlagName, node.Bridge),
+	)
+	bridgeCmd.PersistentFlags().StringP(
+		repoFlagName,
+		repoFlagShort,
+		"~/.celestia-bridge",
```

Comment:
This is the default directory for the node's persistent files, right? If we're building something that isn't just a scappy devnet I think this should be a constant instead of a hard-coded string.

Reply:
@adlerjohn, I agree. Is it ok if I do this in a separate PR where I will do some flag changes? This is a bit unrelated to this one.

Reply:
Yeah sure, just track it in an issue.